src/layouts/home-assistant.ts

```diff
@@ -85,6 +87,7 @@ export class HomeAssistantAppEl extends HassElement {
     super.hassConnected();
     // @ts-ignore
     this._loadHassTranslations(this.hass!.language, "state");
+    getTranslation("config", getLocalLanguage());
```

Comment:
We should not load the config translation on every page load, either move the translations or load the config translations when the dialog is opened

Reply:
Definitely. I'll be doing a translation pass tomorrow

Reply:
Done